crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -74,41 +74,43 @@ def shutdown_services_at_non_db_nodes
     NodeObject.find("state:crowbar_upgrade").each do |node|
 
       # Find the nodes with Database role
-      # (we can't look at node.roles, because that list has been reduced to just crowbar-upgrade)
-      node.crowbar["run_list_map"].each do |role, v|
-        db_nodes.push node.name if role == "database-server"
-      end
+      # (we can't look at node.roles, because that list has been reduced to just crowbar-upgrade item)
```

Comment:
Metrics/LineLength: Line is too long. [102/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)